mmv1/third_party/terraform/services/cloudquotas/data_source_google_cloud_quotas_quota_infos.go

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
Nit: shouldn't need these lines in this repo

Reply:
Removed.